scripts/pipeline_repo_checkout.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -o errexit
+set -o pipefail
+# set -o nounset  # Some arguments may not be set
+# Uncomment this line to see each command for debugging (careful: this will show secrets!)
+# set -o xtrace
+
```

Comment:
could maybe be explicit about which var(s) can be unset

```suggestion
set -o nounset
# Uncomment this line to see each command for debugging (careful: this will show secrets!)
# set -o xtrace

ORG_GH_TOKEN="${ORG_GH_TOKEN:-}"  # May be unset
```